src/components/TranslationLeaderboard.tsx

```diff
@@ -214,45 +192,116 @@ const TranslationLeaderboard: React.FC<IProps> = ({
+                    </Box>
                   ) : (
-                    <ItemNumber>{idx + 1}</ItemNumber>
+                    <Box w={10} opacity="0.4">
+                      {idx + 1}
+                    </Box>
                   )}
-                  <TextContainer>
-                    <Avatar src={user.avatarUrl} />
-                    <NameContainer>
+                  <Flex
+                    flexDirection="row"
+                    alignItems="center"
+                    mr={8}
+                    overflowWrap="anywhere"
+                  >
+                    <Img
+                      mr={4}
+                      h={{ base: "30px", sm: 10 }}
+                      w={{ base: "30px", sm: 10 }}
+                      borderRadius="50%"
+                      display={{ base: "none", s: "block" }}
+                      src={user.avatarUrl}
+                    />
+                    <Box maxW={{ base: "100px", sm: "none" }}>
                       {user.username}
-                      <Language>{sortedLanguages[0].language.name}</Language>
-                    </NameContainer>
-                  </TextContainer>
+                      <Text m={0} display="block" fontSize="sm" opacity="0.6">
+                        {sortedLanguages[0].language.name}
+                      </Text>
+                    </Box>
+                  </Flex>
                 </Flex>
-                <WordsContainer>
-                  <StyledEmoji mr={"0.5rem"} size={1.5} text={":writing:"} />
+                <Flex minW="20%" flexDirection="row" alignItems="left">
+                  <Emoji
+                    display={{ base: "none", sm: "block" }}
+                    mr={2}
+                    fontSize="2xl"
+                    text={":writing:"}
+                  />
                   {user.totalCosts}
-                </WordsContainer>
-              </Item>
+                </Flex>
+              </Flex>
             )
           })}
-      </Table>
-      <OptionContainer>
-        <Option onClick={filterAmount === 10 ? showMore : showLess}>
-          <OptionText fontSize={"18px"}>
+      </Box>
+      <Flex
+        justifyContent="center"
+        py={0}
+        px={8}
+        mb={8}
+        flexDirection={{ base: "column", lg: "inherit" }}
+        w="full"
+      >
+        <Flex
+          borderRadius="2rem"
+          borderWidth="1px"
+          borderStyle="solid"
+          borderColor="text"
+          color="text"
+          alignItems="center"
+          py={4}
+          px={6}
+          m={2}
+          cursor="pointer"
+          bg="transparent"
+          w={{ base: "full", lg: "initial" }}
+          justifyContent="center"
+          ml={{ base: "0", lg: "2" }}
+          mr={{ base: "0", lg: "2" }}
+          _hover={{
+            color: "primary",
+            borderColor: "primary",
+          }}
+          onClick={filterAmount === 10 ? showMore : showLess}
+        >
```

Comment:
This should be a button. I know it was not a button before but I think we should fix this. I'd understand if you don't want to do that on this PR. I could create a new issue to tackle that refactor, up to you. LMK what do you think.

Reply:
And we could share these styles with the ones we are using on the `RadioCard`.

I have in mind this:
```tsx
const Button = (props) => {
  return (
    <ChakraButton
      borderRadius="2rem"
      borderWidth="1px"
      borderStyle="solid"
      borderColor="text"
      ...
      ...props
    />
  )
}

// on the RadioCard component
const RadioCard = () => {
  ...
  
  return (
    <Button
      as="label"
      {...checkbox}
      ...
  )
}
```